Gordon360/Services/NewsService.cs

```diff
@@ -201,8 +201,8 @@ public StudentNews DeleteNews(int newsID)
 
             if (newsItem.Image != null)
             {
-                string fileName = newsItem.SNID + ".jpg";
-                string imagePath = NewsUploadsPath + fileName;
+                var imagePath = Path.Combine(_webHostEnvironment.ContentRootPath, "browseable", "uploads", Path.GetFileName(newsItem.Image));
+
```

Comment:
You have to use this `Path.Combine(_webHostEnvironment.ContentRootPath, "browseable", "uploads", Path.GetFileName(newsItem.Image))` line in several places. We should extract it a private method (or maybe a computed property), so that it's easier to reuse and can be updated in one place.